examples/05_glm_second_level/plot_oasis.py

```diff
@@ -90,13 +89,16 @@
     columns=["age", "sex", "intercept"],
 )
 
+from matplotlib import pyplot as plt
```

Comment:
don't we also need to catch the ImportError in this example?

Reply:
Or move the `from nilearn import plotting` above this import

Reply:
ha yeah that makes sense: will do that for the examples that import pyplot before nilearn plotting

Reply:
OK the logic got a bit more complicated:
- when an example use pyplot, then there a try import block at the top
- when an example use pyplot AND some nilearn.plotting function I hoisted the import nilearn.plotting at the top of the example
- when this was not possible (because isort putting the import matplotlib.pyplot before the nilearn.plotting imports) then I added a try import block